package.json

```diff
@@ -26,7 +26,8 @@
         "cordova-android": ">=4.0.0"
       },
       "2.0.0": {
-        "cordova": ">100"
+        "cordova": ">100",
```

Comment:
What the `"cordova": ">100"` does is to prevent the install of the plugin in case we do a major release and forget to adjust the engines properly, as it will prevent the install, we will notice and update the engines in a new patch release.
What we usually do is to move it to the next major, in this case 3.0.0, but considering that 2.0.0 will be the last one, this line can just be removed.